### ISSUE TITLE
Friendly error message upon user accidentally invoke `Docker build` upon `blimp up`

### ISSUE THREAD

**Issue by rizdaprasetya** [kelda/blimp#35]
Nice product you have here 👍 . Here is some feedback from my personal use:

**Is your feature request related to a problem? Please describe.**
When user accidentally include `build` command on `docker-compose.yml` e.g. step to reproduce:

1. Clone the official example `git clone https://github.com/kelda/node-todo && cd node-todo`
2. Edit `docker-compose.yml`, include a `build` command, like this:
```yml
version: '3'
services:
  web:
    # for example I accidentally include build command here:
    build: .
    entrypoint:
      - "/usr/src/app/node_modules/.bin/nodemon"
      - "/usr/src/app/server.js"
    ports:
      - "8080:8080"
    depends_on:
      - mongo
    volumes:
      - "./app:/usr/src/app/app"
  mongo:
    image: "mongo"
    ports:
      - "27017:27017"

```
3. save the file
4. run `blimp up`

User will get the following error message which might not be descriptive enough for user to fix the issue:
```
➜  node-todo git:(master) ✗ blimp up
Booting cloud sandbox
Building image for web:
[Error] Get help at https://kelda.io/blimp/docs/help/
build web: start build: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
```

**Describe the solution you'd like**
- Catch that error message, and tell user to not avoid docker `build` command in the `docker-compose.yml` file.
OR
- read the `docker-compose.yml` file and if `build` command is detected, refuse to run and display proper err msg.

or maybe somekind of messaging `looks like you are trying to invoke Docker 'build' command, this command is not currently supported in Blimp's cloud container. Please check your 'docker-compose.yml' file and remove the 'build' command`.

**Additional context**
User can easily accidentally include `build` command within `docker-compose.yml` because they think **Blimp** is ready-to-use drop-in-replacement for **Docker Compose** (as you describe it). And lots of users are having `build` command within their `docker-compose.yml`.

If they run `blimp up` and it didn't work, they may think your product is broken or not working as described. They may drop off. So this will be good improvement for new user who might not understand anything of your product. 

Well it took me a while to figure out what went wrong too.

**Comment by kklin** [comment]
Hi @rizdaprasetya, thanks for the feedback and detailed info!

We’re actually in the process of implementing in cluster builds (https://github.com/kelda/blimp/pull/34), which will improve the UX of this a lot. With that change, the build field will work whether or not you have docker installed locally.

In the meantime, you need to have Docker running locally (whether natively on Linux or via Docker Desktop for windows and Mac) in order to build.

**Comment by rizdaprasetya** [comment]
Ah I see just realized it was trying to connect to local Docker (yes I had docker stopped at that time), I thought it was trying to build on cloud/cluster. Ok got it.

Nice to know cluster builds is on the way, that will do. Ok then, feel free to close this issue if you think it is no longer relevant 👍

**Comment by kklin** [comment]
I'll keep it open for now, and close it once the feature is merged. Thanks and please open issues for anything else you run into!

**Comment by kklin** [comment]
Hi @rizdaprasetya, we just released the cloud builds feature. Here's the announcement from slack in case you missed it.

Please give it a try!

---

Hi everyone! We’ve just released in cluster Docker builds :tada:

With this, you can truly uninstall docker and just use blimp. If your compose file has build fields, blimp will ship the build context into our cluster, and build there.

Just upgrade your CLI to use it!

Some notes:
We still use the local docker daemon by default, and fallback to cloud builds if it doesn’t exist (or if --remote-build is set)
Upgrading your CLI will cause any previously built containers to restart. This will only happen the first time you run blimp up.

https://asciinema.org/a/B0rr5UNXYsqp6BEusX2iK5eOu

**Comment by rizdaprasetya** [comment]
Wonderful, and good job on this well communicated answers on handling Github issue/feedbacks!
Will try soon. Thanks 👍
